routers/routes/routes.go

```diff
@@ -466,6 +466,9 @@ func RegisterRoutes(m *macaron.Macaron) {
 				m.Post("/title", repo.UpdateIssueTitle)
 				m.Post("/content", repo.UpdateIssueContent)
 				m.Post("/watch", repo.IssueWatch)
+				m.Post("/add_time", repo.AddTimeManual)
```

Comment:
I would replace `/stop` with `/toggle` because it do not stop the stopwatch it will start or stop depending on the state.